lib/helper/Puppeteer.js

```diff
@@ -1299,6 +1299,27 @@ class Puppeteer extends Helper {
     return equals(`all elements (${locator}) to have attributes ${JSON.stringify(attributes)}`).assert(chunked.length, elemAmount);
   }
 
+  /**
+   * {{> ../webapi/dragSlider }}
+   */
+  async dragSlider(locator, offsetX = 0) {
+    const src = await this._locate(locator);
+    assertElementExists(src, locator, 'Slider Element');
+
+    // Note: Using private api ._clickablePoint becaues the .BoundingBox does not take into account iframe offsets!
```

Comment:
@PeterNgTr "because" should be "because", or?

Reply:
fixed